Tests/E2E/MultiConcentratorTests.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tests.E2E
+{
+    using System;
+    using System.Globalization;
+    using System.Threading.Tasks;
+    using Xunit;
+    using XunitRetryHelper;
+
+    // Tests multi-concentrator scenarios
+    [Collection(Constants.TestCollectionName)] // run in serial
+    [Trait("Category", "SkipWhenLiveUnitTesting")]
+    public sealed class MultiConcentratorTests : IntegrationTestBaseCi
+    {
+        public MultiConcentratorTests(IntegrationTestFixtureCi testFixture)
+            : base(testFixture)
+        {
+        }
+
+        [RetryFact]
+        public async Task Test_Concentrator_Deduplication_OTAA()
+        {
+            var device = TestFixtureCi.GetDeviceByPropertyName("Device31_OTAA");
+            LogTestStart(device);
+
+            await ArduinoDevice.setDeviceModeAsync(LoRaArduinoSerial._device_mode_t.LWOTAA);
+            await ArduinoDevice.setIdAsync(device.DevAddr, device.DeviceID, device.AppEUI);
+            await ArduinoDevice.setKeyAsync(device.NwkSKey, device.AppSKey, device.AppKey);
+
+            await ArduinoDevice.SetupLora(TestFixtureCi.Configuration);
+
+            var joinSucceeded = await ArduinoDevice.setOTAAJoinAsyncWithRetry(LoRaArduinoSerial._otaa_join_cmd_t.JOIN, 20000, 5);
+            Assert.True(joinSucceeded, "Join failed");
+
+            var logMsg = $"Duplicate message received from station with EUI";
```

Comment:
Shall we make this a constant and re-use from `ConcentratorDeduplication` and here? This way if someone adjusts the log message the test won't fail unnecessarily.

Reply:
Unfortunately we can't assert on the exact same message here as we don't know the station EUI. Removed the '$'

Reply:
Changed to use a constant